WPFWijkagent/WPFWijkagent/Scraper.cs

```diff
@@ -16,7 +13,7 @@ public class Scraper
     {
         public Offence Offence { get; set; }
 
-        SearchTweetsParameters SearchParameters;
+        readonly SearchTweetsParameters _searchParameters;
```

Comment:
maakt dit niet de werking van de scraper kapot gezien hij later pas gezet word...?

Reply:
readonly kan als er in de constructor wordt gezet. Het gaf iig geen errors maar als er op andere plekken deze attribuut wordt aangepast moet dit inderdaad weer weg